src/main/java/org/testng/annotations/BeforeMethod.java

```diff
@@ -40,6 +40,8 @@
    */
   public String[] dependsOnMethods() default {};
 
+  public String[] onlyForGroups() default {};
```

Comment:
Can you please add some java docs to this attribute, that explains what this attribute can be used for ?